git_machete/client.py

```diff
@@ -2356,7 +2363,10 @@ def create_github_pr(
         print(fmt(f'{ok_str}, see `{pr.html_url}`'))
 
         milestone_path: str = self.__git.get_main_git_subpath('info', 'milestone')
-        milestone: str = utils.slurp_file_or_empty(milestone_path).strip()
```

Comment:
Same reason, to easily check if both the case with existing and non-existent file is covered with tests